apps/wallet/src/ui/app/hooks/useGetValidators.ts

```diff
@@ -0,0 +1,106 @@
+// Copyright (c) Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+import { isSuiObject, isSuiMoveObject } from '@mysten/sui.js';
+import BigNumber from 'bignumber.js';
+import { useMemo } from 'react';
+
+import { useGetObject } from './useGetObject';
+
+//TODO: Remove when available on SDK, types should come from the SDK
+import type {
+    Validator,
+    ValidatorState,
+} from '_app/staking/home/ValidatorDataTypes';
+
+export const VALIDATORS_OBJECT_ID = '0x05';
```

Comment:
Can we either move this to the SDK or share it internally within the app (e.g. pending delegations)?